### PR TITLE
contact styling markI

### PR DESCRIPTION
**Pull request by Guitarninja1**
template contact
template message
template intouch
my message
my connect
my intouch

### CODE REVIEW

**Comment by kalm42** [comment]
# Contact
Radial gradient
line height
font weight

# Send us a message
Description block
input padding
' *'
text area non-adjustable

# Always be in touch
linear gradient
title in all caps
left pad input
label 'I accept...'
left pad I accept
Terms should be link

File: components/InTouch.js [hunk 1]
```diff
@@ -1,11 +1,34 @@
...
+				Subscribe if you want always to be known about all the news and
+				available propositions.
+			</p>
+			<div className={styles.items}>
+				<label htmlFor='email' className='sr-only'>
+					email
+				</label>
+				<input
+					type='email'
+					id='email'
```
Comment (kalm42) [inline components/InTouch.js:17]: You should be careful of generic ids.

File: components/InTouch.js [hunk 2]
```diff
@@ -1,11 +1,34 @@
...
+					className={styles.emailinput}
+					placeholder='Enter Your E-mail'
+				/>
+				<div className={styles.container}>
+					<label htmlFor='checkbox' className='sr-only' />
```
Comment (kalm42) [inline components/InTouch.js:22]: Your label has no content.

File: components/InTouch.js [hunk 3]
```diff
@@ -1,11 +1,34 @@
...
+					<input
+						type='checkbox'
+						id='checkbox'
+						className={styles.checkbox}
+					></input>
+					<p className={styles.label}>I accept all</p>
+					<p className={styles.terms}>Terms and Conditions</p>
```
Comment (kalm42) [inline components/InTouch.js:29]: lol, that's fancy inline p tags. Don't do that. lol label > text / link

File: components/Message.js [hunk 4]
```diff
@@ -1,17 +1,46 @@
...
+					id='name'
+					placeholder='Name'
+					className={styles.content}
+				/>
+				<label htmlFor='email' className='sr-only'>
+					Email
+				</label>
+				<input
+					type='email'
+					id='email'
```
Comment (kalm42) [inline components/Message.js:22]: This 'id' conflicts with InTouch

File: components/Message.module.css [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+	padding-right: 15px;
+}
+.content {
+	background: transparent;
+	color: #989cac;
+	border: solid rgba(153, 153, 153, 0.05);
+	font-family: Mulish;
+	font-size: 14px;
+	min-width: 97%;
+	background-color: transparent;
```
Comment (kalm42) [inline components/Message.module.css:23]: Consider 'none'.

File: views/contact.module.css [hunk 6]
```diff
@@ -0,0 +1,53 @@
+.image {
+}
```
Comment (kalm42) [inline views/contact.module.css:2]: Empty

File: views/contact.module.css [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+.phone,
```
Comment (kalm42) [inline views/contact.module.css:3]: Not needed with module find an abstract name that covers both.

File: views/contact.module.css [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+.address {
+	display: grid;
+	grid-template-columns: 100px 1fr;
+}
+.content1 {
```
Comment (kalm42) [inline views/contact.module.css:8]: Try for a more descriptive name.

**Review by kalm42 — DISMISSED** [review]

File: components/Breadcrumbs.module.css [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+.unordered {
+	position: relative;
+	display: flex;
+	justify-content: flex-end;
+	margin: -15px 0 0 0;
+	padding: 0 15px 0 0;
+	list-style: none;
+	align-items: center;
+}
+.list,
```
[the remark this answers is not in the document]
Comment (Guitarninja1) [reply components/Breadcrumbs.module.css:15]: I had to separate them to do ::after on .home

File: components/Breadcrumbs.module.css [hunk 10]
```diff
@@ -0,0 +1,23 @@
+.home::after {
+	content: '–';
+	opacity: 0.5;
+	margin: 0 25px;
+}
+.unordered {
+	position: relative;
+	display: flex;
+	justify-content: flex-end;
+	margin: -15px 0 0 0;
```
[the remark this answers is not in the document]
Comment (Guitarninja1) [reply components/Breadcrumbs.module.css:10]: I'm having trouble adjusting the offending element. Could use your help

**Comment by Guitarninja1** [comment]
template contact
template message
template intouch
Screen Shot 2021-08-18 at 10 27 47 AM
Screen Shot 2021-08-18 at 10 27 10 AM
Screen Shot 2021-08-18 at 10 25 04 AM

**Comment by Guitarninja1** [comment]
template intouch
Screen Shot 2021-08-18 at 11 53 12 AM

File: components/Message.js [hunk 11]
```diff
@@ -1,17 +1,57 @@
...
-				<textarea defaultValue='Comment' />
-				<input type='submit' value='Submit' className='message--submit' />
+		<div className={styles.wrapper}>
+			<h2 className={styles.title}>Send Us A Message</h2>
+			<p className={styles.text}>
+				Pariatur voluptate occaecat nostrud est exercitation. Culpa culpa
+				laboris excepteur officia ullamco sunt nisi velit culpa.
+			</p>
+			<form className={styles.items}>
+				<label htmlFor='name' className='sr-only'>
```
Comment (kalm42) [inline components/Message.js:12]: The htmlFor doesn't match the id of the input it's labeling.

File: components/Message.js [hunk 12]
```diff
@@ -1,17 +1,57 @@
...
+					Email
+				</label>
+				<input
+					type='message--email'
+					id='email'
+					placeholder='E-mail *'
+					className={styles.content}
+					required='true'
+				/>
+				<label htmlFor='tel' className='sr-only'>
```
Comment (kalm42) [inline components/Message.js:32]: htmlfor

File: components/Message.js [hunk 13]
```diff
@@ -1,17 +1,57 @@
...
+					Name
+				</label>
+				<input
+					type='text'
+					id='message--name'
+					placeholder='Name *'
+					className={styles.content}
+					required='true'
+				/>
+				<label htmlFor='email' className='sr-only'>
```
Comment (kalm42) [inline components/Message.js:22]: htmlfor

File: components/Message.js [hunk 14]
```diff
@@ -1,17 +1,57 @@
...
+					Email
+				</label>
+				<input
+					type='message--email'
```
Comment (kalm42) [inline components/Message.js:26]: You updated the type not the. 'message--email' is not a valid input type.

File: components/Message.js [hunk 15]
```diff
@@ -1,17 +1,57 @@
...
+					id='message--tel'
+					placeholder='Phone #'
+					className={styles.content}
+				/>
+				<textarea
+					rows='5'
+					cols='30'
+					placeholder='Comment'
+					className={styles.textarea}
+				/>
```
Comment (kalm42) [inline components/Message.js:46]: Inputs need a label

File: views/Contact.module.css [hunk 16]
```diff
@@ -0,0 +1,58 @@
...
+	font-size: 16px;
+	font-family: Mulish;
+	font-weight: normal;
+	font-style: normal;
+	line-height: 1.1;
+	color: #989cac;
+	margin: 30px 0 10px 0;
+}
+.address {
+}
```
Comment (kalm42) [inline views/Contact.module.css:40]: Empty

**Review by kalm42 — CHANGES_REQUESTED** [review]

File: components/InTouch.js [hunk 17]
```diff
@@ -1,11 +1,40 @@
...
+					placeholder='Enter Your E-mail'
+				/>
+				<div className={styles.container}>
+					<input
+						type='checkbox'
+						id='checkbox'
+						className={styles.checkbox}
+					></input>
+					<label htmlFor='checkbox'>
+						<div className={styles.foo}>
```
Comment (kalm42) [inline components/InTouch.js:29]: I suggest changing this from a div to a p.

File: components/InTouch.module.css [hunk 18]
```diff
@@ -0,0 +1,84 @@
...
+	color: #ff455b;
+	margin: 0;
+	font-size: 14px;
+}
+.checkbox {
+	appearance: none;
+	background-color: transparent;
+	border: 2px solid rgba(153, 153, 153, 0.2);
+	box-shadow: 0 1px 2px rgba(0, 0, 0, 0.05),
+		inset 0px -15px 10px -12px rgba(0, 0, 0, 0.05);
```
Comment (kalm42) [inline components/InTouch.module.css:66]: Can you see this?

File: components/InTouch.module.css [hunk 19]
```diff
@@ -0,0 +1,84 @@
...
+	display: flex;
+	align-items: center;
+	font-size: 14px;
+	margin-top: 12px;
+}
+.checkbox,
+.label {
+	margin-right: 5px;
+}
+.foo {
```
Comment (kalm42) [inline components/InTouch.module.css:82]: Is there a better name?

**Review by kalm42 — APPROVED** [review]

**Review by kalm42 — APPROVED** [review]

**Pull request merged**